android/app/src/main/res/values-hu/strings.xml

```diff
@@ -621,8 +621,8 @@
 	<string name="place_description_title">A hely ismertetése</string>
 	<!-- this text will be shown in application notification preferences opposite checkbox which enable/disable downloader notifications. Devices on Android 8+ are affected. -->
 	<string name="notification_channel_downloader">Kártyák letöltése</string>
-	<!-- Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit -->
-	<string name="pref_tts_speedcams_auto">Figyelmeztetés ha van kockázat a sebesség korlátozás megsértéséről</string>
+	<!-- "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit -->
+	<string name="pref_tts_speedcams_auto">Figyelmeztetés a sebességhatár átlépésekor</string>
```

Comment:
```suggestion
	<string name="pref_tts_speedcams_auto">Figyelmeztetés gyorshajtáskor</string>
```

Reply:
@d4f5409d: Please check...

![ios_speed_cam_prefs_hu3](https://github.com/organicmaps/organicmaps/assets/25888296/4b0bab7a-6e99-4051-9b6f-4c7fe13b4621)

Reply:
"Soha sem figyelmeztet" is the correct form, a typo ;)

Reply:
@d4f5409d: Here you go...

![ios_speed_cam_prefs_hu4](https://github.com/organicmaps/organicmaps/assets/25888296/4c9ada90-5dd3-4e2f-8673-e4a78f210db1)